asserts/gpgkeypairmgr.go

```diff
@@ -183,3 +223,83 @@ func (gkm *GPGKeypairManager) sign(fingerprint string, content []byte) ([]byte,
 	}
 	return out, nil
 }
+
+type gpgKeypairInfo struct {
+	pubKey      PublicKey
+	fingerprint string
+}
+
+func (gkm *GPGKeypairManager) findByName(name string) (*gpgKeypairInfo, error) {
+	stop := errors.New("stop marker")
+	var hit gpgKeypairInfo
```

Comment:
I mean that there is already lots of code using the term "KeypairManager", and so "KeypairInfo" should surely be capitalised consistently with that.  I don't much care which way round it goes, of course, so maybe a future branch could rename KeypairManager etc. to KeyPairManager?